src/holder/verifiable_presentation_builder.rs

```diff
@@ -0,0 +1,164 @@
+use std::time::{SystemTime, UNIX_EPOCH};
+
+use serde::{Deserialize, Serialize};
+use ssi_claims::jwt::{VerifiableCredential, VerifiablePresentation};
```

Comment:
I think there is a confusion here, probably because of the lack of documentation in ssi I'll admit. Those two types are wrappers around the JWT registered claims `vc` and `vp` to be used with `ssi_claims::jwt::ClaimSet`.

If you want to represent VCs and VPs, the appropriate types are defined in `ssi_claims::vc`.